src/resources/packages/classy/fields/EventDetails/EventDetails.tsx

```diff
@@ -25,6 +26,7 @@ export default function EventDetails( props: FieldProps ) {
 
 	const [ description, setDescription ] = useState< string >( postContent || '' );
 	const [ eventUrlValue, setEventUrlValue ] = useState< string >( eventUrlMeta );
+	const [ isUrlInvalid, setIsUrlInvalid ] = useState< boolean >( false );
```

Comment:
The starting value should be based on `eventUrlMeta`, right?

Reply:
Can you change the const name to `isUrlValid`? Easier to use in functions following human preference for positive logic.